src/ItemNavigator/ItemNavigator.js

```diff
@@ -37,23 +38,23 @@ export default class ItemNavigator extends LitElement {
     };
   }
 
-
   constructor() {
     /** TODO: Request BookModel.js
-     * Request BookNavigator.js
-     * Show loading spinner
-     * When JS assets loaded:
-     * - render book-navigator component
-     */
+    * Request BookNavigator.js
+    * Show loading spinner
+    * When JS assets loaded:
+    * - render book-navigator component
+    */
     super();
-    this.baseHost = "archive.org";
+    this.baseHost = 'archive.org';
```

Comment:
Why so many linting changes here? I don't think we enforce single vs double quotes in our eslint; let's remove the linting changes to keep the history tidy. If we decide to pick one, then we can apply it across the whole codebase in one fell swoop.

Reply:
nav refresh from pb